README.md

```diff
@@ -117,6 +217,8 @@ A production deployment can improve upon that by using some other key delivery m
 
 The following diagram illustrates how the PMEM-CSI driver performs dynamic volume provisioning in Kubernetes:
 ![sequence diagram](/docs/images/sequence/pmem-csi-sequence-diagram.png)
+<!-- TODO: update the diagram to show how ephemeral volumes get created and deleted -->
```

Comment:
Strictly speaking, each persistency mode would need its own diagram. My proposal is to show it only for the ephemeral case, because that is a) the most important one and b) a bit unusual in how it handles PVC/PV.